vaadin-popover-flow-parent/vaadin-popover-flow/src/test/java/com/vaadin/flow/component/popover/PopoverTest.java

```diff
@@ -65,4 +69,62 @@ public void setPosition_getPosition() {
     public void defaultPosition_equalsNull() {
         Assert.assertEquals(null, popover.getPosition());
     }
+
+    @Test
+    public void getTrigger_defaultValue_click() {
+        Assert.assertEquals(Set.of(PopoverTrigger.CLICK), popover.getTrigger());
+    }
+
+    @Test
+    public void getTriggerProperty_defaultValue_click() {
```

Comment:
Nit: IMO we could move raw property assertions into the respective tests that checks the property getter

Reply:
Updated tests.